test/state-time.test.ts

```diff
@@ -43,4 +43,13 @@ describe("dust state at multiple time points", () => {
         expect(state.getState(0, 5)).toEqual(t5.getState(0));
         expect(state.getState(3, 5)).toEqual(t5.getState(3));
     });
+
+    it("can extract a specific trace", () => {
```

Comment:
There is some behaviour not under test here - that a vector from `getTrace` is a new copy, whereas `viewTrace` allows editing in place.